insta2/src/components/CommentSection/CommentSection.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+// import './CommentSection.css'
+import PropTypes from 'prop-types';
+import CommentForm from '../CommentForm';
+
+class CommentSection extends React.Component {
+   constructor(props) {
+       super(props)
+       console.log(props)
+       this.state = {
+           comments: props.comments
+        }
+    }
+    addNewComment = (event, indexNum, commentText) => {
+        console.log(this.state)
+        const copiedComments = [...this.state.comments]
+        const arrayWithNewComment = copiedComments.push(commentText)
```

Comment:
to add in the new commentText after spreading the state, you could add it to that spread array by comma separating them like this, [...this.state.comments, commentText]